conf/ocsci/rosa.yaml

```diff
@@ -0,0 +1,34 @@
+---
+ENV_DATA:
+  platform: 'rosa'
+  deployment_type: 'managed'
+  addon_name: 'ocs-converged'
+  configs: 'aws,stage'
+  worker_instance_type: 'm5.2xlarge'
+  worker_replicas: 3
+  master_replicas: 3
+  # size of storage cluster, default is 1 TiB
+  size: '1'
+  #skip_ocp_deployment: true
+  # notification_email_0: MAIL USED FOR SENDGRID
+
+# AUTH:
+#  openshiftdedicated:
+#    token: OCM TOKEN KEY PLACEHOLDER
+
+DEPLOYMENT:
+  force_download_ocm_cli: True
+  ocm_cli_version: '0.1.46'
+  force_download_rosa_cli: True
+  rosa_cli_version: '1.1.2'
+  region: "us-east-2"
+  addon_name: "ocs-converged"
+  force_download_installer: True
+  force_download_client: True
+  ocp_version: "4.9.4"
```

Comment:
You should not set those values, this is set by --ocp-version , OCP_VERSION in jenkins